src/common/wallet-state.ts

```diff
@@ -290,7 +268,6 @@ function useWalletState(initialState?: Partial<WalletStateParams>): WalletData {
   const accounts = getOrElse((): Account[] => [])(accountsOption)
 
   const getOverviewProps = (): Overview => {
-    const transactions = getOrElse((): Transaction[] => [])(transactionsOption)
```

Comment:
Shouldn't we remove transactions from `getOverviewProps` then and just export them directly from wallet state?

Reply:
You're probably right that it is more than enough to distinguish `LOADED` state. I didn't want to touch such things in this PR though

Reply:
Yeah, nevermind, it's done in Mice's PR :) https://github.com/input-output-hk/mantis-wallet/pull/64/files#diff-1b88ee2da726519fc36f436b52425c965edc4af5a5d2c398f68825f59b2b7f9bL292